### ISSUE TITLE
Should "Summaries available" change from core to supplemental?

### ISSUE THREAD

**Issue by julierawe** [w3c/wcag3#551]
In the January 2026 WCAG 3 survey, @sydneycolemanfromgoogle commented:

- Summaries Available: It will be tricky to define a min amount of words/characters that would need summarization, and testing steps would be hard to define in a reliable way. I feel like there are so many tools for summarization nowadays, that this should be supplemental, allowing users to use summarization tools they prefer.

Sydney's comment raises an interesting question:
Should WCAG 3 consider having a pair of requirements about summaries?
- **(Core requirement) Text content summarizable** would ensure text content is available to/detectable by user agents/summarization tools.
- **(Supplemental requirement) Text content summaries provided** would require content authors to provide a summary.  

As we consider these possible changes, we need to look at other sections of WCAG 3 to see if adding a new requirement would create unnecessary overlap, such as with "Text detectable" in the Text-to-speech section.
